app/src/components/common/TmPage.vue

```diff
@@ -22,14 +22,16 @@
       :subtitle="emptySubtitle"
     />
 
-    <slot v-else-if="!loading && !empty"></slot>
+    <template v-else-if="!loading && !empty">
+      <slot></slot>
+      <TmDataLoading v-if="!loading && loadingMore" class="loading-more" />
```

Comment:
we don't need this class "loading-more" the styles should go in `TmDataLoading`

Reply:
we actually do. It needs to appear on the bottom when it is the `loadingMore`case

Reply:
ah no sorry, the styles were simplified. OK, doing then